app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
Yeah, so it would be better to split hosts and subnet related resources into different controllers so we *can* use the helpers and we're more likely to pick up fixes and features from Foreman.

This doesn't have anything to do with the UI, as far as I can tell.